sdk/src/main/java/com/supenta/flitchio/sdk/FlitchioController.java

```diff
@@ -394,19 +394,23 @@ public void onDestroy() {
              */
             try {
                 flitchioService.removeClientInfo(authToken);
-
             } catch (RemoteException e) {
                 FlitchioLog.e("Unexpected error: could not notify Flitchio Manager about " +
                         "this controller termination.");
 
-            } catch (NullPointerException | IllegalArgumentException e) {
+            } catch (NullPointerException e) {
                 FlitchioLog.w("Warning: binding to Flitchio Manager not yet effective.");
             }
 
             /*
              * UNBIND
              */
-            context.unbindService(serviceConnection);
+            try {
+                context.unbindService(serviceConnection);
+            } catch (IllegalArgumentException e){
```

Comment:
My formatter in my laptop is indeed a bit fucked, I need to import the settings from the office computer.